pygameMenu/menu.py

```diff
@@ -276,17 +301,36 @@ def __init__(self,
                         (self._posx + self._width, self._posy + self._height),
                         (self._posx, self._posy + self._height)
                         ]
-        self._draw_regionx = draw_region_x
-        self._draw_regiony = draw_region_y
+        self._draw_regionx = draw_region_x  # type: int
+        self._draw_regiony = draw_region_y  # type: int
 
         # Option position
-        self._opt_posx = int(self._width * (self._draw_regionx / 100.0)) + self._posx
         self._opt_posy = int(self._height * (self._draw_regiony / 100.0)) + self._posy
-        self._widget_align = widget_alignment
+
+        # Columns support
+        if columns > 1:
+            if column_weights is None:
+                column_weights = tuple(1 for _ in range(columns))
+            s = float(sum(column_weights[:columns]))
+            cumulative = 0
+            self._column_posx = []

```

Comment:
Strange to have attributes `self._column_posx` and `self._column_widths` defined only if several columns exist.
Shouldn't the case of a column be threatened like several columns?

Reply:
I've refactored the menu, now by default the number of rows is 1e6 and there's only 1 column.